### PR TITLE
Update the ascend npu test repository link

### PR DESCRIPTION
**Pull request by xuedinge233**
### Summary

Transfer the ascend npu test of liger-kernel from the individual warehouse to the Ascend organization.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -499,8 +499,8 @@ loss.backward()
                 </a>
             </div>
             <div style="display: block;">
-                <a href="https://github.com/xuedinge233/Liger-Kernel/actions/workflows/ascend_npu_ci.yml">
-                    <img src="https://github.com/xuedinge233/Liger-Kernel/actions/workflows/ascend_npu_ci.yml/badge.svg?branch=main" alt="Build">
+                <a href="https://github.com/Ascend/Ascend-CI/actions/workflows/liger_kernel.yml">
+                    <img src="https://github.com/Ascend/Ascend-CI/actions/workflows/liger_kernel.yml/badge.svg?branch=main" alt="Build">
```
Comment (zheliuyu) [inline README.md]: Thanks for switching this to the official Ascend-CI workflow.

Quick question: the GitHub badge will show "Liger kernel test on Ascend NPU" instead of "Ascend NPU" — was that intentional? Just wondering if we should keep the shorter label to match the other hardware badges.

截屏2026-08-29 11 21 22

Comment (xuedinge233) [reply README.md]: The Ascend-CI repository also has ci work for other tasks. It's not easy to distinguish them without adding the task names.

Comment (zheliuyu) [reply README.md]: Try this: 

```
<a href="https://github.com/Ascend/Ascend-CI/actions/workflows/liger_kernel.yml">
    <img src="https://img.shields.io/github/actions/workflow/status/Ascend/Ascend-CI/liger_kernel.yml?branch=main&label=Ascend%20NPU" alt="Ascend NPU">
</a>

```

<a href="https://github.com/Ascend/Ascend-CI/actions/workflows/liger_kernel.yml">
    Ascend NPU
</a>

Comment (xuedinge233) [reply README.md]: I have completed the revisions, but it still looks somewhat different from the others.
image

Comment (xuedinge233) [reply README.md]: I have made overall changes to the display style of ci status, and now they are unified.

image

**Comment by zheliuyu** [comment]
@Tcc0403 Could you please help decide which display option is acceptable?

**Comment by Tcc0403** [comment]
The latest one is fine. Is it good to merge?

**Comment by zheliuyu** [comment]
@Tcc0403  Yes, the latest version looks good to me. Thanks for the updates — good to merge.

**Review by Tcc0403 — APPROVED** [review]
